### ISSUE TITLE
[Discussion] Naming Conventions

### ISSUE THREAD

**Issue by markusphil** [OpenTechSchool-Leipzig/poll_app#38]
## The Problem
I'm no expert on surveys (and also not a native english speaker) but I got the feeling, that that naming of the app and it's components is a little misleading regarding conventions for the words *poll*, *survey* and *evaluation form*.

Here is my impression for the conceptions of these terms:
I guess that the general expactation for a *poll* is that it only contains one question (and is embeded on a website/forum). While *surveys* are concieved to include multiple questions and are common tools for science (e.g. social sciences and psychology)  and economy (e.g. market surveys). An *evaluation form* is more specialized and bound to a specific event and date.

The guiding idea behind the webapp was to provide a tool to evaluate workshops and track the feedback over multiple worshops or events. But we also wanted it to be more general and be usable for all kind of questionaires.

Having that in mind the name *poll* feels very missleading for the purpose of the application.

## Solutions
Now I have to ideas working on that:

### 1. Generalze
Give the app a more general name like "feedback" and add options for three variants:
- poll: only one questions
- survey: multiple questions
- evaluation: multiple questions and fixed date

### 2. Specialize
Keeping our focus straight on event evaluation and adjust our namings regarding that.

---

As I'm actually not sure about these asumptions about polls and surveys I'd really appreciate your feedback and your ideas about how we should continue with the application!

**Comment by Twissi** [comment]
Yes, I think your assumptions are right. For me a poll is a one question survey which usually has given answers (no free forms). I like the idea to rename it feedback, because it does collect feedback.
